README.md

```diff
@@ -190,5 +190,15 @@ A collection of experiments and research to learn about what makes an Alaveteli
         <strong>No.</strong> Not enough traffic to make an impact.
       </td>
     </tr>
+ <tr>
+      <td>
+        <a href="experiments/what-factors-caused-WDTK-to-grow.md">
```

Comment:
house style - filenames should be lower case (file should be renamed as well but I can't work out how to comment that \*sigh\*)